Pod/Classes/OTReference.h

```diff
@@ -0,0 +1,45 @@
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
My (incomplete) understanding is that this is needed to make Swift interoperability work, in particular regarding the `nullable` annotations. While I think it's probably *not* good to require Cocoa APIs that are only available in iOS9, etc, AFAIK it *is* ok to assume a certain version of Xcode. If that turns out to be an invalid assumption, then we can revisit this at that time.

The ideal thing would be to have `github.com/opentracing/opentracing-swift`, of course, but that's "future work".